Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json

```diff
@@ -17,6 +17,14 @@
       {
         "name": "inhand-right",
         "directions": 4
+      },
+      {
+        "name": "equipped-BACKPACK",
```

Comment:
add information about the sprites to the "copyright" line above
for example
"equipped sprites made by Hoodie42 for Space Station 14"

Reply:
Thanks, i added it next to the other copyright !